bench/CMakeLists.txt

```diff
@@ -32,3 +32,5 @@ add_executable(hashmap_bench
   ../hashmap.h
   main.c
 )
+
+target_link_libraries(hashmap_bench m)
```

Comment:
Why is this needed?

Reply:
No way to access the sqrt() function on GCC or Clang without the math library linked in.  I don't know whether CMake does the right thing for Visual Studio though.  I have no way to test on that platform.

Reply:
Oh interesting - its a benchmark specific problem. I'll file a bug ta!